app/routes/info.py

```
@@ -1319,16 +1383,38 @@ async def event_generator():
                 done = asyncio.Event()
                 final_result = {"valid": False, "message": "Unknown error"}
 
-                async def run_verify(path=path):
+                bound = await tool.verify_timeout(path)
+
+                async def run_verify(path=path, bound=bound):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `run_verify`

Add return type annotation: `None`

(ANN202)